src/Mailbox.php

```diff
@@ -247,4 +247,25 @@ public function getThread(): array
 
         return false !== $tree ? $tree : [];
     }
+
+    /**
+     * Bulk move messages.
+     *
+     * @param array|MessageIteratorInterface|string $numbers Message numbers
+     * @param MailboxInterface                      $mailbox Destination Mailbox to move the messages to
+     *
+     * @return bool true on success
+     */
+    public function move($numbers, MailboxInterface $mailbox): bool
+    {
+        if ($numbers instanceof MessageIterator) {
+            $numbers = $numbers->getArrayCopy();
+        }
+
+        if (\is_array($numbers)) {
+            $numbers = \implode(',', $numbers);
+        }
+
+        return \imap_mail_move($this->resource->getStream(), (string) $numbers, $mailbox->getEncodedName(), \CP_UID);
```

Comment:
I would like to maintain similarity with:
https://github.com/ddeboer/imap/blob/f90d03fa1a1497a00ffecb278330edfaf2c596bc/src/Message.php#L260-L262
Please remove any return and throw the Exception.

Reply:
I actually was about to implement it with an exception, but decided against, as the other bulk commands (_clearFlag_ and _setFlag_) use a boolean return too.